Sources/MongoDBVapor/ExtendedJSON+Content.swift

```diff
@@ -0,0 +1,24 @@
+import MongoSwift
+import Vapor
+
+// These extension enable users to encode/decode their Content types to/from JSON using `ExtendedJSONEncoder` and
+// `ExtendedJSONDecoder`. This is useful in cases where users are using one of our custom BSON types in their
+// structs which does not currently support encoding/decoding via `JSONEncoder`/`JSONDecoder`.
+
+extension ExtendedJSONEncoder: ContentEncoder {
```

Comment:
this file was conveniently a straight copy paste from 
https://github.com/vapor/vapor/blob/master/Sources/Vapor/Content/JSONCoders%2BContent.swift with "Extended" inserted a couple times 🙂